src/state/optionsSlice.ts

```diff
@@ -1,13 +1,29 @@
 import { createSlice } from '@reduxjs/toolkit';
 import { TooltipEventType } from '../util/types';
+import { TooltipPayloadSearcher } from './tooltipSlice';
 
 export type ChartOptions = {
   chartName?: string;
   defaultTooltipEventType?: TooltipEventType;
   validateTooltipEventTypes?: ReadonlyArray<TooltipEventType>;
+  // Should this instead be a property of a graphical item? Do we want to mix items with different data types in one chart?
```

Comment:
If we ever want to display Line on top of Treemap or Sankey charts, this will become important. Until then we can leave it like this I think.